unittests/wasm_tests.cpp

```diff
@@ -64,6 +67,12 @@ struct provereset {
    }
 };
 
+namespace {
+struct old_wasm_tester : tester {
```

Comment:
The modified test cases were using `TESTER` before which may have been `validating_tester`. Now they are effectively only using `tester`. So it is possible we may have lost coverage of important cases from those modified unit tests.

Reply:
It's a pain, because validating_tester doesn't have a constructor that takes a setup_policy.  I believe that #8697 includes some changes that make it easier.